docker-compose.yml

```diff
@@ -144,7 +153,7 @@ services:
     depends_on:
       - api
       - dev-tls
-
+  
```

Comment:
```suggestion

```

Reply:
> 

Why would you remove \n ? It would make the docker-compose less readable